blocks/community/layout/community-layout.tsx

```diff
@@ -52,44 +59,51 @@ export const CommunityLayout: FC<CommunityLayoutProps> = ({title, children}) =>
                 <meta name="viewport" content="width=device-width" />
             </Head>
 
-            <GlobalHeader
-                currentUrl={COMMUNITY_URL}
-                currentTitle={COMMUNITY_TITLE}
-                productWebUrl={releasesData.latest.url}
-                hasSearch={true}
-                onSearchClick={onSearch}
-            />
+            <StickyHeader>
```

Comment:
I'm not sure that the whole header has to be sticky.
It is contradicted current behavior on the [Mobile](https://kotlinlang.org/lp/mobile/), [Server-Side](https://kotlinlang.org/lp/server-side/), and [Teach](https://kotlinlang.org/education/) pages.

Reply:
Oh, you're right, will fix